x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/saved_object_mappings.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ElasticsearchMappingOf } from '../../../utils/typed_elasticsearch_mappings';
+import { IRuleStatusAttributes } from './types';
+
+export const ruleStatusSavedObjectType = 'siem-detection-engine-rule-status';
+
+export const ruleStatusSavedObjectMappings: {
+  [ruleStatusSavedObjectType]: ElasticsearchMappingOf<IRuleStatusAttributes>;
+} = {
+  [ruleStatusSavedObjectType]: {
+    properties: {
+      alertId: {
+        type: 'text',
+      },
```

Comment:
I think you want `keyword` here instead of `text` since its an id?

Reply:
Yes definitely. I changed it to text when I wasn't sure how I was going to search on it and never changed it to keyword. Thanks for the catch.